app/views/posts/show.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Single post page</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Post Details</title>
+</head>
+<body>
+<div class="post-card">
+      <p class="comt-likes"><strong>Comments:</strong> <%= @post.comments.count %> &nbsp;&nbsp; <strong>Likes:</strong> <%= @post.likes.count %></p>
+  <h5><%= @post.title %> by <%= @user.name %></h5>
+  <p><%= @post.text %></p>
+</div>
+<div class="post-cmts">
+  <h2>comments:</h2>
+  <% @post.return_five_most_recent_comments.each do |comment| %>
+    <p><strong><%= comment.author.name %>:</strong> <%= comment.text %></p>
+  <% end %>
```

Comment:
- [ ] [optional] @tomdan-ai Great work implementing the change requested, however, the previous reviewer was wrong, you only need to display five message in posts index page , but display all messages in the show page as shown in the wireframes below
![image](https://github.com/tomdan-ai/Gem-Blog/assets/61408860/577464b2-2810-48e9-a65d-6359c0a350c9)